style/style.css

```diff
@@ -0,0 +1,183 @@
+.container{
+    max-width: 1000px;
+    margin: 0 auto;
+    padding: 0 10px;
+}
+.page-header{
+    padding: 30px 0;
+}
+@media screen and (min-width: 1200px){
+    .page-header__wrapper{
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+    }
+}
+@media screen and (min-width: 0) and (max-width: 1200px) {
+    .page-header__wrapper{
+        text-align: center;
+    }
+    .page-logo{
+        display: block;
+        margin-bottom: 15px;
+    }
+}
+.slider{
+    text-align: center;
+}
+.slider > img{
+    width: 100%;
+}
+.box{
+    border: 1px solid black;
+}
+.box img{
+    width: 100%;
+}
+.box__content{
+    padding: 10px 0 0 0;
+}
+.box__content > p{
+    padding: 5px;
+}
+.grid-columns-item{
+    margin: 30px 0;
+}
+.box-buttons{
+    display: flex;
+    align-items: center;
+}
+.button-desc{
+    background-color: #b9b9b9;
+    flex-basis: 50%;
+    text-align: center;
+    padding: 10px;
+}
+.button-desc > a{
+    text-decoration: none;
+    color: #000;
+}
+.button-buy{
+    background-color: #14881d;
+    flex-basis: 50%;
+    text-align: center;
+    padding: 10px;
+}
+.button-buy > a{
+    text-decoration: none;
+    color: #fff;
+}
+@media screen and (min-width: 768px){
+    .grid-columns{
+        display: flex;
+        flex-wrap: wrap;
+        gap: 30px;
```

Comment:
jest okej, ale jeśli myslimy o wiekszym wsparciu to przy okazji pracy z flexem, warto "gap-a" zastapic tym trickiem z paddingami ktory wam pokazalem